### DISCUSSION TITLE
Is PHP 8.0 docker image stable and ready to use?

### DISCUSSION THREAD

**Discussion by virtgen** [markshust/docker-magento#542] [General]
Hello Mark,

There is a git comment 'New image "8.0-fpm-develop" available for testing purposes' for PHP 8.0 image and you still use PHP 7.4 in docker-compose.yml.

What is the current status of PHP 8.0 in docker-magento rep? Is it stable already to replace 7.4 with 8.0?

Thank you!

**Comment by rswitala** [comment]
Magento 2.4.3 does not support php 8.
Support will be added in version 2.4.4

**Comment by markshust** [comment]
I just pushed up a new `8.0-fpm-develop` image a few days back, which is basically a clone of the 7.x images (without ionCube, because that is not yet available for PHP 8). If you downloaded it already, you can delete it and re-pull it from Docker Hub.

The `platform-health` branch on Magento's repo supports PHP 8, and I verified this image works with it. The `2.4-develop` branch should also now work with PHP 8.
